src/lib/learn-client/api/bookmark/__tests__/create-bookmark.test.ts

```diff
@@ -0,0 +1,68 @@
+import { ApiBookmark } from 'lib/learn-client/api/api-types'
+import { post } from 'lib/learn-client'
+import { createBookmark } from '../create-bookmark'
+import { BOOKMARK_API_ROUTE } from '..'
+
+const mockBookmark: ApiBookmark = {
+	tutorial_id: 'test-tutorial-1',
+	tutorial: {} as $TSFixMe,
+	user_id: 'test-user',
+}
+
+jest.mock('lib/learn-client', () => {
```

Comment:
Don't you want to mock just the `createBookmark` function? Or what's the thought behind mocking the entire client?

Reply:
These were set up to do [partial mocks](https://jestjs.io/docs/mock-functions#mocking-partials) of each method in `lib/learn-client`, so only the method used by the bookmark method being tested is what was getting mocked.

I went ahead and changed this in https://github.com/hashicorp/dev-portal/pull/828/commits/c27723485a2696d86fec44eebbcddb9bbe5b0d8a though, so the import is different and we can instead use `spyOn` and `restoreAllMocks`. Both make less code in the end. 😊